db/dcrsqlite/blocksretrieval_test.go

```diff
@@ -48,3 +48,45 @@ func TestEmptyDBRetrieveBlockFeeInfo(t *testing.T) {
 			cmp.Diff(*result, defaultChartsData))
 	}
 }
+
+func TestEmptyDBGetBestBlockHash(t *testing.T) {
+	testutil.BindCurrentTestSetup(t)
+	db := ObtainReusableEmptyDB()
+	str := db.GetBestBlockHash()
+	if str != "" {
+		// Open question: Should it really be the empty string?
+		// Maybe error instead to avoid confusion?
+		testutil.ReportTestFailed(
+			"GetBestBlockHash() failed: %v",
+			str)
+	}
+}
+
+func TestEmptyDBGetBestBlockHeight(t *testing.T) {
+	testutil.BindCurrentTestSetup(t)
+	db := ObtainReusableEmptyDB()
+	h := db.GetBestBlockHeight()
+	if h != -1 {
+		testutil.ReportTestFailed(
+			"db.GetBestBlockHeight() is %v,"+
+				" should be -1",
+			h)
```

Comment:
This wrapping isn't making it any easier to read.  This whole function call can be on one line in my book

Reply:
Also, use `%d` for integers please.